Resources/Prototypes/Loadouts/Miscellaneous/trinkets.yml

```diff
@@ -29,6 +29,27 @@
     back:
     - PlushieSpaceLizard
 
+# CD: Lunchbox
+- type: loadout # CD
+  id: LunchboxBlue
+  equipment: LunchboxBlue
+
+- type: startingGear # CD
+  id: LunchboxBlue
+  storage:
+    back:
+    - LunchboxBlue
+
+- type: loadout # CD
+  id: LunchboxYellow
+  equipment: LunchboxYellow
+
+- type: startingGear # CD
+  id: LunchboxYellow
+  storage:
+    back:
+    - LunchboxYellow
+
```

Comment:
Can you move this to it's own file under _CD?

Reply:
oops

Reply:
Resolved as of 21c635d8fd87c159edc8c194d345fd317068a91e